apps/console/src/extensions/components/authenticators/fido/quick-start.tsx

```diff
@@ -120,6 +123,119 @@ const FIDOQuickStart: FunctionComponent<FIDOQuickStartPropsInterface> = (
+                                        <li>
+                                            <Trans
+                                                i18nKey={ "extensions:develop.identityProviders.fido." +
+                                                "quickStart.steps.configureParameters.content.steps.3" }
+                                            >
+                                                Navigate to the <strong>Settings</strong> tab.
+                                            </Trans>
+                                        </li>
+                                    </ol>
+                                </Grid.Column>
+                                <Grid.Row>
+                                    <Grid.Column>
+                                        <GenericIcon 
+                                            inline
+                                            transparent
+                                            icon={ ConfigureParametersIllustration }
+                                            size="huge"/>
+                                    </Grid.Column>
+                                </Grid.Row>
+                            </Grid.Row>
+                            <Grid.Row>
+                                <Grid.Column>
+                                    <Typography variant="h6">
+                                        { t("extensions:develop.identityProviders.fido.quickStart.steps." +
+                                        "configureParameters.content.parameters.progressiveEnrollment.label") } 
+                                    </Typography>
+                                    { t("extensions:develop.identityProviders.fido.quickStart." + 
+                                        "steps.configureParameters.content.parameters." + 
+                                        "progressiveEnrollment.description") }
+                                </Grid.Column>
+                            </Grid.Row>
+                            <Grid.Row>
+                                <Grid.Column>
+                                    <Alert severity="info">
+                                        <span>
+                                            <Trans
+                                                i18nKey={ "extensions:develop.identityProviders.fido." +
+                                                "quickStart.steps.configureParameters.content.parameters." +
+                                                "progressiveEnrollment.note" }
+                                            >
+                                                If Passkey is set as a first factor, the following 
+                                                <strong> adaptive script</strong> should be added under 
+                                                the <strong>Sign-In-Method</strong> tab of the application. 
+                                                This script is used to verify the user&apos;s identity 
```

Comment:
This script is added automatically with the template-based passkey setup and is used to verify the user's identity before enrolling passkeys. However, if you're configuring passkeys without a template, remember to add the script manually.

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/4750/commits/63c0c3ad085943ac55238c33cf3756343f7c88d1